tools/deploy_translatable_strings.sh

```diff
@@ -80,7 +80,7 @@ git add setup.py
 git add requirements-dev.txt constraints.txt
 
 # Commit and push the changes.
-git commit -m "Automated documentation update to add .po files from meta-qiskit" -m "skip ci" -m "Commit: $GITHUB_SHA" -m "Github Actions Run: https://github.com/Qiskit/qiskit/runs/$GITHUB_RUN_NUMBER"
+git commit -m "Automated documentation update to add .po files from qiskit" -m "skip ci" -m "Commit: $GITHUB_SHA" -m "Github Actions Run: https://github.com/Qiskit/qiskit/runs/$GITHUB_RUN_NUMBER"
```

Comment:
```suggestion
git commit -m "Automated documentation update to add .po files from qiskit" -m "skip ci" -m "Commit: $GITHUB_SHA" -m "Github Actions Run: $GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID"
```

I _think_ this is the correct mapping of all the variable names (and correction to the URL) that makes the link actually work and independent of what GitHub instance and repository the action's deployed on (public vs enterprise, etc)